storage/innobase/fil/fil0fil.cc

```diff
@@ -1959,11 +1959,11 @@ Allocates and builds a file name from a path, a table or tablespace name
 and a suffix. The string must be freed by caller with ut_free().
 @param[in] path NULL or the directory path or the full path and filename.
 @param[in] name NULL if path is full, or Table/Tablespace name
-@param[in] suffix NULL or the file extention to use.
+@param[in] ext NULL or the file extention to use.
```

Comment:
"extension" would be the grammatically correct word. In the function declaration, the parameter is called `suffix`, which is a little confusing. Maybe it should be renamed there as well?

Reply:
Done